pkg/sys/tc.go

```diff
@@ -3,20 +3,15 @@
 package sys
 
 import (
-	"unsafe"
-
 	"golang.org/x/sys/unix"
 )
 
+/* This function is currently unused.
```

Comment:
Remove

Reply:
That works for me. I left it in case you had a good reason for writing it in the first place.